### ISSUE TITLE
Paper 1.20.6 i can`t fix nick name players and other

### ISSUE THREAD

**Issue by Fragmer2** [kloppie74/BetterChat#1]
![image](https://github.com/user-attachments/assets/fb4fa78d-3966-4eb4-adb9-54365d995ada)
I also can't edit the text Chat game and 
![image](https://github.com/user-attachments/assets/2e1820f4-fabb-499e-8c36-7cb1d4eaad76)

**Comment by kloppie74** [comment]
Hi, for %player_name% to work, you need to use placeholderAPI plugin!

**Comment by Fragmer2** [comment]
yes now it work, but how i can edit [ChatGame] change words and color?
